### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/calyxos/seedvault/) for CalyxOS/Seedvault.

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by chirayudesai** [comment]
@mikeNG Did we decide on just translating / merged into android11?

That'd make master behind android11, not fun.

**Comment by mikeNG** [comment]
> @mikeNG Did we decide on just translating / merged into android11?
> 
> That'd make master behind android11, not fun.

That's the only way to get correct translations for android11 when using weblate (only supports one branch), since there might be new/changed/removed strings in master branch that are not in android11

File: app/src/main/res/values-my/strings.xml [hunk 1]
```diff
@@ -1,2 +1,74 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources></resources>
\ No newline at end of file
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="notification_error_text">ကူးယူမှုမအောင်မြင်ပါ။</string>
+    <string name="notification_error_title">မှားယွင်းနေပါသည်</string>
+    <string name="notification_error_channel_title">သတိပေးချက်အမှား</string>
+    <string name="notification_failed_title">မအောင်မြင်ပါ</string>
+    <string name="notification_success_text">အပ်ပလီကေးရှင်းများကူးထားပြီပါပြီ</string>
```
Comment (chirayudesai) [inline app/src/main/res/values-my/strings.xml:7]: values-my/strings.xml:7: This definition does not require arguments

File: app/src/main/res/values-my/strings.xml [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+    <string name="notification_success_title">Backup ပြီးဆုံးသွားပါပြီ</string>
+    <string name="notification_backup_already_running">Back upလုပ်နေပါသည်</string>
+    <string name="notification_title">Back up ပြန်ဆောင်ရွက်နေပါသည်။</string>
+    <string name="notification_channel_title">အရန်ကူးအကြောင်းကြားစာ</string>
+    <string name="recovery_code_error_checksum_word">သင်ကုဒ်မှားယွင်းနေပါသည်။ပြန်လည်စစ်ဆေးပြီပြန်ရိုက်ထည့်ပါ</string>
+    <string name="recovery_code_error_invalid_word">စကားလုံးမှားယွင်းနေပါသည်။</string>
```
Comment (chirayudesai) [inline app/src/main/res/values-my/strings.xml:13]: values-my/strings.xml:13: This definition does not require arguments

File: app/src/main/res/values-my/strings.xml [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+    <string name="settings_backup_apk_dialog_disable">အက်ပ်အရန်ကူးခြင်းကိုပိတ်ပါ</string>
+    <string name="settings_backup_apk_dialog_cancel">ပယ်ဖြတ်ခြင်း ။</string>
+    <string name="settings_backup_apk_dialog_message">ပိတ်ထားသောအက်ပလီကေးရှင်းသည်အက်ပ်ဒေတာကိုအရံသိမ်းလိမ့်မည်။ သို့သော်၎င်းသည်အလိုအလျောက်ပြန်လည်ရရှိမည်မဟုတ်ပါ။
+\n
+\n\"Automatic Restore\" ကိုဖွင့်ထားစဉ်သင်၏အက်ပ်များအားလုံးကိုကိုယ်တိုင်ထည့်သွင်းရန်လိုအပ်သည် ။</string>
+    <string name="settings_backup_apk_dialog_title">အက်ပလီကေးရှင်းအရန်ကူးခြင်းကိုအမှန်ပိတ်ထားသလား ။</string>
+    <string name="settings_backup_apk_summary">အက်ပ်များကိုသူတို့ကိုယ်တိုင်အရန်ကူးပါ။ ဒီလိုမှမဟုတ်ရင် app data များသာအရန်သင့်သိမ်းဆည်းလိမ့်မယ် ။</string>
+    <string name="settings_backup_apk_title">အက်ပ်ဒေတာအရံကူးခြင်း ။</string>
+    <string name="settings_category_app_data_backup">အက်ပ်ဒေတာအရံကူးခြင်း</string>
+    <string name="settings_auto_restore_summary_usb">သင်၏% 1 $ s ကိုဤအလုပ်မလုပ်ရန်ချိတ်ဆက်ရန်လိုအပ်သည်.</string>
```
Comment (chirayudesai) [inline app/src/main/res/values-my/strings.xml:61]: values-my/strings.xml:61: This definition does not require arguments

**Review by chirayudesai — CHANGES_REQUESTED** [review]
In that case, let's merge this.

I commented on the strings the CI blew up on, haven't looked at the others.

**Review by chirayudesai — APPROVED** [review]
Did some minor fixes in local commits.

**Pull request merged**